dataflow/conftest.py

```diff
@@ -850,6 +856,7 @@ def dataflow_extensible_template_run(
             f"--project={project}",
             f"--region={region}",
             f"--staging-location=gs://{bucket_name}/staging",
+            f"--temp-location=gs://{bucket_name}/staging",
```

Comment:
Some environments are reporting this parameter doesn't exist.

Reply:
some but not all? these should both be available as per https://cloud.google.com/dataflow/docs/guides/templates/configuring-flex-templates#locations